lib/Raven/Serializable.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/*
+ * Copyright 2012 Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+/**
+ * This helper is based on code from Facebook's Phabricator project
+ *
+ *   https://github.com/facebook/phabricator
+ *
+ * Specifically, it is an adaptation of the PhutilReadableSerializer class.
+ *
+ * @package raven
+ */
+
+interface Raven_Serializable
+{
+    /**
+     * The return value would be how the object is represented in the Sentry
+     * stack trace.
+     * Can be any primitive value.
+     * In case of array, it must contains only primitive values.
+     *
+     * @return string|numeric|array
```

Comment:
https://scrutinizer-ci.com/g/getsentry/sentry-php/inspections/cf2e7d62-67af-4828-9eec-bef7e2dcafaa/issues/files/lib/Raven/ReprSerializer.php?status=new&orderField=path&order=asc&honorSelectedPaths=0

As I can see toSentry may return array. But serializeValue must return "null|boolean|double|integer|string". No arrays allowed. All arrays must be serialized as well.